tests/js/client/shell/shell-replication-vpack-cluster.js

```diff
@@ -0,0 +1,131 @@
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global assertEqual, assertNotEqual, assertTrue, assertFalse, arango, print */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / DISCLAIMER
+// /
+// / Copyright 2014-2024 ArangoDB GmbH, Cologne, Germany
+// / Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+// /
+// / Licensed under the Business Source License 1.1 (the "License");
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     https://github.com/arangodb/arangodb/blob/devel/LICENSE
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is ArangoDB GmbH, Cologne, Germany
+// /
+// / @author Max Neunhöffer
+// //////////////////////////////////////////////////////////////////////////////
+
+const jsunity = require('jsunity');
+const internal = require('internal');
+const arangodb = require('@arangodb');
+const db = arangodb.db;
+let IM = global.instanceManager;
+const { waitForShardsInSync } = require("@arangodb/test-helper");
+
+function replicationSuite() {
+  'use strict';
+  const cn = 'UnitTestsCollection';
+
+  return {
+
+    setUp: function () {
+      db._drop(cn);
+      db._create(cn, { numberOfShards: 1, replicationFactor: 1 });
+    },
+
+    tearDown: function () {
+      arango.forceJson(true);
+      db._drop(cn);
+    },
+    
+    testReplication: function() {
```

Comment:
I would add the comment what exactly this test is testing since it is hard to say without being introduced to the issue itself

Reply:
Fair point, explanation added.

Reply:
That is a nice explanation!